app/src/main/res/values/strings.xml

```diff
@@ -1,8 +1,22 @@
 <resources>
-    <string name="app_name">GeoQuiz</string>
-    <string name="true_button">True</string>
-    <string name="false_button">False</string>
-    <string name="question_australia">Canberra is the capital of Australia.</string>
-    <string name="correct_toast">Correct!</string>
-    <string name="incorrect_toast">Incorrect</string>
+        <string name="app_name">Test</string>
+        <string name="true_button">True</string>
+        <string name="false_button">False</string>
+        <string name="question_australia">Canberra is the capital of Australia.</string>
+        <string name="correct_toast">Correct!</string>
+        <string name="incorrect_toast">Incorrect</string>
+        <string name="next_button">Next</string>
+        <string name="previous_button">Previous</string>
+        <string name="question_oceans">The Pacific Ocean is larger than
+the Atlantic Ocean.</string>
+        <string name="question_mideast">The Suez Canal connects the Red Sea
+and the Indian Ocean.</string>
+        <string name="question_africa">The source of the Nile River is in Egypt.</string>
+        <string name="question_americas">The Amazon River is the longest river
+in the Americas.</string>
+        <string name="question_asia">Lake Baikal is the world\'s oldest and deepest
+freshwater lake.</string>
+        <string name="previous_button_image">True</string>
+        <string name="next_button_image">True</string>
```

Comment:
The string names should represent the values, not the usages.
There should be a single string with value "True" that's shared across all usages.

Refactoring example:
<string name="true">True</string>